### ISSUE TITLE
Some articles have HTML artefacts in bookmark view

### ISSUE THREAD

**Issue by ISNIT0** [kiwix/kiwix-android#537]
**Is it a ?**

- [x] Bug Report

- [ ] Feature Request

### Step 1: Are you ready to report?
   - Your issue may already be reported! Please search on the [issue track](../) before creating one.
   - [x] I have tested with the last (dev) version of the software.  ( If not, and if possible, run your test again with the most recent version available.)

### Step 2: What is your environment?
  * version of the software you use : Commit 149088ee0f86cc0f6dfceb7d717f00e96b7717c4
  * device / operating system : Android
  
### Step 3: Describe the problem:

  * What do you obtain? A copy of the error message or a screenshot is always useful.
![screenshot_2018-03-01-14-11-24](https://user-images.githubusercontent.com/5173131/36849747-c01ef8a6-1d5c-11e8-86b3-ae0d0e7491fa.png)

  * When does this occur? 

#### Steps to reproduce:

  1. Download "Bollywood" ZIM (103.8MB)
  2. Search for "Battle for Bittora" (a side-bug is that the search item is in italics too)
  3. Bookmark the article
  4. View Bookmark lists
  
  * What should be the behaviour from your point of view? How do you expect the service to work?
Should either:
- Strip all HTML tags from search results and Bookmark View
- Render HTML tags as should be (`<i>` should be italics)

### Step 4 (optional) : Other Comments:
  * What is the context of this ticket? If not obvious, explain why you need to do this.
 
  * If you have an idea about the technical background of the ticket, please share it.
  
  
#### Feature Description

Please describe the feature you want to add to the project.

**Comment by harshvardhan-takawale** [comment]
@ISNIT0 Are you working on this issue? I would like to give it a try if you are not.

**Comment by ISNIT0** [comment]
@harshvardhan-takawale Go for it!
The artefacts also seem to show in the left slide-out menu 👍

**Comment by harshvardhan-takawale** [comment]
@brijeshshah13 @srv-twry can you assign me the issue?

**Comment by mhutti1** [comment]
GitHub is weird about who can be assigned. We can do some people but others need to accept an invite which should just have been sent to you.

**Comment by RajaVamsi11** [comment]
@mhutti1 I tried assigning this issue for @harshvardhan-takawale  but the name is not present in list.Why?

**Comment by mhutti1** [comment]
@RajaVamsi11 See my comment just above yours where I explain it in detail.

**Comment by harshvardhan-takawale** [comment]
@ISNIT0 I am facing a problem in reproducing the bug, it might be another bug. I downloaded the Bollywood Zim and searched "Battle of Bittora" and "Dliwale Dulaniya le jayange", I can see the names below the search bar but when I tap on I it doesn't open but takes me to Bollywood's homepage. @brijeshshah13 @srv-twry @RajaVamsi11 can you guys confirm the behavior?

**Comment by ISNIT0** [comment]
Hmm, I don't get that at all - what device are you using?

**Comment by harshvardhan-takawale** [comment]
@ISNIT0 Moto G4 Plus.

**Comment by harshvardhan-takawale** [comment]
@ISNIT0 
![20180302_140325](https://user-images.githubusercontent.com/28982436/36894408-38813898-1e31-11e8-8d40-d671a1af980e.gif)

**Comment by ISNIT0** [comment]
I think I originally found it by clicking through links - try finding it through links I guess?

**Comment by harshvardhan-takawale** [comment]
@ISNIT0 Yes found it through the links. So there is no problem in the zim file I guess. So must be a but in the app? What do you think?
